lms/djangoapps/mobile_api/tests/test_course_info_views.py

```diff
@@ -521,3 +524,77 @@ def verify_certificate(self, response, mock_certificate_downloadable_status):
         mock_certificate_downloadable_status.assert_called_once()
         certificate_url = 'https://test_certificate_url'
         assert response.data['certificate'] == {'url': certificate_url}
+
+    @patch('lms.djangoapps.mobile_api.course_info.utils.certificate_downloadable_status')
+    def test_course_not_started(self, mock_certificate_downloadable_status):
+        """ Test course data whic is not started yet """
+
+        certificate_url = 'https://test_certificate_url'
+        mock_certificate_downloadable_status.return_value = {
+            'is_downloadable': True,
+            'download_url': certificate_url,
+        }
+        now = datetime.now(utc)
+        course_not_started = CourseFactory.create(
+            mobile_available=True,
+            static_asset_path="needed_for_split",
+            start=now + timedelta(days=5),
+        )
+
+        url = reverse('course-enrollment-details', kwargs={
+            'api_version': 'v1',
+            'course_id': course_not_started.id
+        })
+
+        response = self.client.get(path=url)
+        assert response.status_code == 200
+        assert response.data['id'] == str(course_not_started.id)
+
+        self.verify_course_access_details(response)
+
+    @patch('lms.djangoapps.mobile_api.course_info.utils.certificate_downloadable_status')
+    def test_course_closed(self, mock_certificate_downloadable_status):
+        """ Test course data whic is not started yet """
```

Comment:
Nit: Is this bad copy-paste? Should the description for this test be different than the above unit test?